backends/jms/pom.xml

```diff
@@ -41,6 +41,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+
+        <!-- Avoid warnings during build -->
+        <dependency>
```

Comment:
To be precise, it's provided and optional: https://github.com/hibernate/hibernate-search/blob/master/pom.xml#L549
I don't think there is a better scope in maven that match what we need for our use case.